tools/kyma-installer/kyma.Dockerfile

```diff
@@ -1,6 +1,4 @@
-ARG INSTALLER_VERSION="98e02519"
-ARG INSTALLER_DIR=eu.gcr.io/kyma-project
-FROM $INSTALLER_DIR/kyma-operator:$INSTALLER_VERSION
+FROM eu.gcr.io/kyma-project/kyma-operator:master-98e02519
```

Comment:
This image is not present in gcr: https://console.cloud.google.com/gcr/images/kyma-project/EU/kyma-operator?gcrImageListsize=30
But `eu.gcr.io/kyma-project/kyma-operator:98e02519` is, so I think we should change that.